angular-workspace/projects/ni/spright-angular/README.md

```diff
@@ -0,0 +1,13 @@
+<div align="center">
+    <p><b>ni | spright | angular</b></p>
+</div>
+
+# Spright Angular
+
+[![NPM Version](https://img.shields.io/npm/v/@ni/spright-angular.svg)](https://www.npmjs.com/package/@ni/spright-angular)
+
+Spright components for [Angular](https://angular.io) applications
```

Comment:
Let's link to the nimble-angular README too since it has the getting started instructions